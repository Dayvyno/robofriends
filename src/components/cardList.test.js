import {shallow} from 'enzyme';
import React from 'react'
import CardList from './cardList';

it("Testing the card component", ()=>{
  const mockRobots = [
    {
      id:1,
      name: "John Snow",
      email: "johnsnow@gmail.com",
      key: 1
    }
  ]
  expect(shallow(<CardList robots={mockRobots} />)).toMatchSnapshot()
})