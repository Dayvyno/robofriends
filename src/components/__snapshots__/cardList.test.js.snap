// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing the card component 1`] = `
<div>
  <div
    className="container"
  >
    <Card
      email="johnsnow@gmail.com"
      id={1}
      key="1"
      name="John Snow"
    />
  </div>
</div>
`;
